.eslintrc.json

```
@@ -0,0 +1,32 @@
+{
+    "parser": "@typescript-eslint/parser",
+    "extends": [
+      "eslint:recommended",
+      "plugin:react/recommended",
+      "plugin:@typescript-eslint/recommended",
+      "plugin:prettier/recommended",
+      "airbnb",
+      "airbnb/hooks"
```

```suggestion
      "airbnb",
      "airbnb/hooks",
      "plugin:prettier/recommended"
```

> Add plugin:prettier/recommended as the last item in the extends array in your .eslintrc* config file, so that eslint-config-prettier has the opportunity to override other configs:

[공식 문서](https://github.com/prettier/eslint-plugin-prettier?tab=readme-ov-file#configuration-legacy-eslintrc)에서 extends의 가장 마지막 item으로 `plugin:prettier/recommended`를 추가해달라고 설명되어 있기에 수정이 필요해보여요.